src/main/kotlin/com/kotlin/boardproject/domain/post/PostController.kt

```diff
@@ -1,25 +1,25 @@
-package com.kotlin.boardproject.controller
+package com.kotlin.boardproject.domain.post
 
-import com.kotlin.boardproject.auth.LoginUser
-import com.kotlin.boardproject.common.enums.NormalType
-import com.kotlin.boardproject.common.util.log
-import com.kotlin.boardproject.dto.comment.CommentsByPostIdResponseDto
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.dto.post.BlackPostRequestDto
-import com.kotlin.boardproject.dto.post.BlackPostResponseDto
-import com.kotlin.boardproject.dto.post.CancelLikePostResponseDto
-import com.kotlin.boardproject.dto.post.CancelScrapPostResponseDto
-import com.kotlin.boardproject.dto.post.LikePostResponseDto
-import com.kotlin.boardproject.dto.post.ScrapPostResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.CreateNormalPostRequestDto
-import com.kotlin.boardproject.dto.post.normalpost.CreateNormalPostResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.DeleteNormalPostResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.EditNormalPostRequestDto
-import com.kotlin.boardproject.dto.post.normalpost.EditNormalPostResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.FindNormalPostByQueryRequestDto
-import com.kotlin.boardproject.dto.post.normalpost.NormalPostByQueryResponseDto
-import com.kotlin.boardproject.dto.post.normalpost.OneNormalPostResponseDto
-import com.kotlin.boardproject.service.PostService
+import com.kotlin.boardproject.global.annotation.LoginUser
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end